.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,3 +1,5 @@
+# Feature request template
```

Comment:
Again, [md041](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md041---first-line-in-file-should-be-a-top-level-header)